lib/Dialect/TritonGPU/IR/LinearLayoutConversions.cpp

```diff
@@ -300,9 +301,9 @@ AMDMfmaEncodingAttr::toLinearLayout(ArrayRef<int64_t> shape) const {
   // https://github.com/ROCm/amd_matrix_instruction_calculator can print the
   // register and lane layout for mfma instructions.
 
-  // We use the order from fastest varying to slowest varying. So each base
-  // vector is a tuple of values mapping to matrix C's (N, M[, B]) indices.
-  SmallVector<unsigned> order = triton::gpu::getOrder(*this);
+  // We use the threadOrder from fastest varying to slowest varying. So each
```

Comment:
This pull request is a mix of things. Can you extract the changes to mfma parts into its own separate pull request so we can keep this one purely for wmma?

Reply:
I am going to make new PRs related to different aspects, working on this PR is very inconvenient.
Will keep this one as a prototype.